### PR TITLE
Add correction support for conference

### LINKED ISSUE #704: Add CORRECTION support to group chats
`tox_conference_send_message` only supports NORMAL and ACTION messages at the moment.

### PR DESCRIPTION
**Pull request by Diadlo**
Fix #704

---
This change is [Reviewable](https://reviewable.io/reviews/toktok/c-toxcore/705)

### CODE REVIEW

**Comment by iphydf** [comment]
:lgtm_strong:

---

Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/c-toxcore/705#-:-L3IFEzH3qyEkQz-4vys:b-jbsf8z)*

File: toxcore/tox.c [hunk 1]
```diff
@@ -1300,8 +1300,10 @@ bool tox_conference_send_message(Tox *tox, uint32_t conference_number, TOX_MESSA
 
     if (type == TOX_MESSAGE_TYPE_NORMAL) {
         ret = group_message_send((Group_Chats *)m->conferences_object, conference_number, message, length);
-    } else {
+    } else if (type == TOX_MESSAGE_TYPE_ACTION) {
```
Comment (iphydf) [inline toxcore/tox.c]: Remove trailing whitespace.

Comment (Diadlo) [reply toxcore/tox.c]: Done.

**Comment by robinlinden** [comment]
:lgtm_strong:

---

Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks pending.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/toktok/c-toxcore/705#-:-L3Ix2ed-dkqVG62JzJO:b-jbsf8z)*

**Pull request merged**